.changelog/016af652b95e472f960ded9a2e976e11.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "016af652-b95e-472f-960d-ed9a2e976e11",
+    "type": "bugfix",
+    "description": "`internal/ini`: Disable noramalization of config profile names",
+    "modules": [
+        "."
+    ]
```

Comment:
Minor nit: s/noramalization/normalization 

Also do you think its more readable to use "core" than "." (not much important as this info is displayed externally)

Reply:
Fixed typo, as indicated in another PR `modules` is used to indicate paths relative to the repository source root where a module is located, and not the name of the module. This intentionally done to keep the code generic enough so that we can use it in the smithy-go repository without special logic. Thus why there is no special treatment or aliasing involved here.